### ISSUE TITLE
Request to change the Accessibility="Internal" to public.

### ISSUE THREAD

**Issue by GSANUGS** [rafabu/SCOM-PKICertificateMP#36]
Hi Refabu,

First of all thank you very much for such a nice MP.

In my environment i am using  PKI Certificate Validation V3  Management Pack. My organization has a different requirement i have to create a new monitor based on the existing monitor "Certificate Lifespan" and the recovery tasks.
The MontiorType and the Recovery Task are defined Accessibility="Internal". Due to this the specific monitor type and recovery task are not accessible outside of the Management Pack. Could you please release a latest version that include the Accessibility="Internal" to public.

Thanks & Regards,
Anu
